src/sql/magic_cmd.py

```diff
@@ -204,6 +207,220 @@ def execute(self, cmd_name="", others="", cell="", local_ns=None):
 
             return report
 
+        elif cmd_name == "explore":
+            parser = CmdParser()
+            parser.add_argument(
+                "-t", "--table", type=str, help="Table name", required=True
+            )
+            args = parser.parse_args(others)
+            # https://stackoverflow.com/questions/38893448/pagination-on-pandas-dataframe-to-html
+
+            # SET CONNECTION
+            import sqlalchemy
+
+            query = f"SELECT * FROM {args.table}"
+            # query = f"SELECT * FROM {args.table} limit 50"
+            # Run query on a given table to fetch first X rows
+            conn = sql.connection.Connection.current
+            # TODO Change this to match custom drivers
+            query = conn._transpile_query(query)
+            rows = conn.session.execute(sqlalchemy.sql.text(query)).fetchall()
```

Comment:
looks like this is fetching the entire table all at once? If the table is large, this will break. One of the points in the acceptance criteria is to fetch one page at a time (e.g., fetch 10, then, if I click on the second page, fetch the next 10). Is this possible with the current architecture?

Reply:
this looks relevant: https://ipywidgets.readthedocs.io/en/stable/examples/Widget%20Low%20Level.html

Reply:
and to clarify: it's not like we don't want to use ipywidgets. if it makes things easier, it's fine to use them. but we need to customize them to our needs.

Reply:
this was just released: https://github.com/widgetti/ipyreact